scengen/generator.py

```diff
@@ -346,3 +345,32 @@ def _resolve_identifiers(input_value: Any, options: dict) -> Any:
                     input_value[key][index] = _get_value_from_field(item, options)
         else:
             input_value[key] = _get_value_from_field(value, options)
+
+
+def _update_series_paths(scenario: dict, options: dict, template_dir: Path) -> None:
+    """
+    Appends relative paths directing to `template_dir` for CSV files defined in `scenario`s `Agents`
+    and (optional) `StringSets`
+    """
+    config_dir = Path(options[CreateOptions.CONFIG]).parent
+    output_dir = Path(options[CreateOptions.DIRECTORY])
+    path_to_append = Path(os.path.relpath(config_dir, start=output_dir), template_dir.parent)
+    for agent in scenario["Agents"]:
+        _replace_timeseries_path_in(agent.get("Attributes", {}), path_to_append)
+    for string_set in scenario.get("StringSets", {}):
+        _replace_timeseries_path_in(scenario["StringSets"][string_set], path_to_append)
+
+
+def _replace_timeseries_path_in(attributes: dict, template_path: Path) -> None:
+    """Recursively modify timeseries path in-place in given `attributes` to link to `template_path`"""
+    for attribute_name, attribute_value in attributes.items():
+        if isinstance(attribute_value, str):
+            if attribute_value.lower().endswith(".csv"):
```

Comment:
Also das geht ist aber auch nicht sooo sicher, weil die Prüfung auf den Attributtyp im Schema fehlt.
Ich würde das so lassen, aber schlage gleichzeitig vor in FAME-Io helper-Methode zur Anpassung von Pfaden einzuführen?